api/v1/flow/flow.proto

```diff
@@ -556,6 +556,10 @@ message FlowFilter {
 
     // trace_id filters flows by trace ID
     repeated string trace_id = 28;
+
+    // cel_expression takes a common expression language (CEL) expression
+    // returning a boolean to determine if the filter matched or not.
+    repeated string cel_expression = 33;
```

Comment:
i thought about where to put this a bit. i mildly prefer putting it as a single string (not array) field inside the `Experimental` section of `GetFlows` request. having this as a repeated string for both allow and deny filters seems a bit confusing 💭

Reply:
I want it in `FlowFilter`. This lets us get it automatically in export filters, and anything else reusing `FlowFilter`. It lets us combine it with other filters too. 

Example: we can't use CEL to easily filter flow source/dest labels, which is much easier to use with existing label filters.

Reply:
... or alternatively just put this filed inside `Experimental` section.